### ISSUE TITLE
Insert in the number format rather than general in excel

### ISSUE THREAD

**Issue by iamalinaji** [pratik227/vue3-json-excel#23]
Hi, 
I was wondering if it's possible to store json values that are numbers with number format in Excel. 
I am using version 1.0.20 and here is what I am trying:
```
          <download-excel :data="json_data" type="xlsx" name="entries.xlsx">
            <v-icon>mdi-export</v-icon>
          </download-excel>

```
And the json_data is:
```
       json_data: [
        {
          Amount : 100,
        },
        {
          Amount: 200,
        },
      ],
```
Unfortunately, Excel recognizes them as general:
<img width="1289" alt="image" src="https://github.com/pratik227/vue3-json-excel/assets/91538770/9f955ba0-26da-4379-bb70-c642d6a50bb0">
Thank you for your help!

**Comment by pratik227** [comment]
https://github.com/pratik227/vue3-json-excel/pull/25

**Comment by pratik227** [comment]
Hi @iamalinaji ,

https://vue-json-excel3.netlify.app/cell-format-xlsx-examples.html

Available in new version please check(**1.0.24**).

Thanks,
Pratik Patel
